Components/Cart/cart.html

```diff
@@ -13,70 +13,78 @@
                 </div>
             </a>
-            <a class="nav-icon-link" href="#">
-                <div class="badge"><img class="header-icon"
-                        src="https://img.icons8.com/external-icongeek26-glyph-icongeek26/38/ffffff/external-cart-essentials-icongeek26-glyph-icongeek26.png" />
+            <a class="nav-icon-link" href="/Components/Cart/cart.html">
+                <div class="badge"><i class="fa fa-shopping-cart fa-2x header-icon " aria-hidden="true"></i>
+                    <div class="number-one">3</div>
                 </div>
             </a>
-            <a class="nav-icon-link" href="#"><img class="header-icon"
-                    src="https://img.icons8.com/material/38/ffffff/user-male-circle--v1.png" /></a>
+            <a class="nav-icon-link" href="/Components/LogIn/login.html"><i class="fa fa-user fa-2x header-icon"
+                    aria-hidden="true"></i></a>
         </nav>
     </div>
     <div class="cart-container">
         <h1 class="header-large">My Cart</h1>
         <div class="item-price-container-row">
-            <div class="cart-card">
-                <img class="horizontal-card-image "
-                    src="/Components/Homepage/HomePage-assets/trending-image-sherlock-cap.jpg" />
-                <div class="image-card-content-col">
-                    <div class="card-content">
-                        <div class=" cart-card-title">
-                            SHERLOCK'S HAT
-                        </div>
-                        <div class="card-content price">
-                            <h2>Rs.500 <strike><span class="light-text">Rs.2000</span></strike> </h2>
-                            <h3>(75% off)</h3>
-                        </div>
-                        <div class="quantity-setting">
-                            <p>Quantity:</p>
+
+            <div class=" horizontal-card  ">
+                <div class="horizontal-card-image-container">
+                    <img class="horizontal-card-image"
+                        src="https://github.com/kotesh-arya/Handy_UI-Component_Library/blob/Fix-cards/Assets/Images/Updated-watch.jpg?raw=true "
+                        alt="horizontal" />
+                </div>
+                <div class="horizontal-card-content">
+                    <div class=" cart-card-title">
+                        <h1 class="header-text header-large">TIMEX SPORT</h1>
+                    </div>
+                    <div class="card-content price">
+                        <h2>Rs.500 <strike><span class="light-text">Rs.2000</span></strike> </h2>
+                        <h3>(75% off)</h3>
+                    </div>
+                    <div class="quantity-setting">
+                        <h4>Quantity:</h4>
+                        <div class="counter-buttons">
                             <button class="btn-counter">-</button>
                             <input class="quantity-input" type="text">
```

Comment:
type is number and set a default value to it.

Reply:
Updated that, Thanks for pointing out it  👍🏻